src/checks/incorrect_delimiter.rs

```diff
@@ -16,7 +16,11 @@ impl Default for IncorrectDelimiterChecker {
 impl Check for IncorrectDelimiterChecker {
     fn run(&mut self, line: LineEntry) -> Option<Warning> {
         let key = line.get_key()?;
-        if key.trim().chars().any(|c| !c.is_alphabetic() && c != '_') {
+        if key
+            .trim()
+            .chars()
+            .any(|c| !c.is_alphabetic() && !c.is_numeric() && c != '_')
```

Comment:
Rust has the method [is_alphanumeric](https://doc.rust-lang.org/std/primitive.char.html#method.is_alphanumeric) which uses these methods under the hood. Can we use it?

Reply:
Sure 👌🏻